challenges-webapp/src/main/webapp/vue-app/challenges/components/ChallengeDrawer.vue

```diff
@@ -228,6 +228,7 @@ export default {
       }
       this.$refs.challengeProgram.broadcast = false;
       this.$refs.challengeProgram.program =  this.challenge.program;
+      this.$refs.challengeProgram.broadcast = true;
```

Comment:
We can remove the line 229 instead of duplicating instruction

Reply:
it is not a duplication it is just enabling then disabling broadcast program change